client/my-sites/hosting/github-authorize-card/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { Button, Card } from '@automattic/components';
+import { translate } from 'i18n-calypso';
+import { useSelector } from 'react-redux';
+import CardHeading from 'calypso/components/card-heading/index';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors/index';
+import iconGitHub from './github.svg';
+
+import './style.scss';
+
+export const GithubAuthorizeCard = () => {
+	const siteId = useSelector( getSelectedSiteId );
+	return (
+		<Card className="github-authorize-card">
+			<img className="github-icon" src={ iconGitHub } alt="" />
+			<CardHeading>{ translate( 'Connect GitHub' ) }</CardHeading>
+			<p>{ translate( 'Connect to a GitHub repository and branch and deploy directly.' ) }</p>
```

Comment:
What do you think about:
`Connect this site to a GitHub repository, choose a branch, and deploy with each push.`

Reply:
Works for me.